sqlmesh/core/console.py

```diff
@@ -441,10 +460,26 @@ def stop_creation_progress(self, success: bool = True) -> None:
         self.environment_naming_info = EnvironmentNamingInfo()
         self.default_catalog = None
 
+    def start_cleanup(self, ignore_ttl: bool) -> bool:
+        if ignore_ttl:
+            self._print(
+                "Are you sure you want to delete all orphaned snapshots regardless of their environment?"
```

Comment:
I don't understand this message. Doesn't `oprhaned` mean they no longer belong to any environments?

Reply:
I've adjusted the wording to make it clearer about what will happen. Is it better?